addons/point_of_sale/migrations/13.0.1.0.1/pre-migration.py

```diff
@@ -14,6 +14,9 @@
         ('account_move', None),
         ('invoice_id', None),
     ],
+    'pos_config': [
+        ('stock_location_id', None),
```

Comment:
No need to do this. This is only done when we fear the field is going to be overwritten. Anyways, we can let it this way, but it complicates the later SQL queries.

Reply:
That was my thinking, but it was failing in CI when it tried to get the legacy column name...

Reply:
Then don't use the legacy column name and use the normal column name 🤔

Reply:
Ok. I was assuming it was always renamed...